pax/config/XENON1T.ini

```diff
@@ -188,12 +188,25 @@ pressure =                            1.94 * bar         # Daily check email 16
 temperature =                         (-95.7 + 273.15) * K    # TE105 on 16 October, described as "top of drift region" here: xenon:xenon1t:dsg:cryogenics:tpc:temperature_sensors
 anode_voltage =                       4.0 * kV           # xenon:xenon1t:aalbers:drift_and_diffusion
 
+# Pe pulse model
+    # Old model:
+    #pe_pulse_model            =           'exponential'
```

Comment:
Yes, it's better to keep the rise time and fall time uncommented so people can switch by changing just one setting, I'll change it.